src/shared/gif/index.js

```diff
@@ -296,7 +299,7 @@ export default class ConverseGif {
         if (!this.frame) return;
         this.frames.push({
             data: this.frame.getImageData(0, 0, this.hdr.width, this.hdr.height),
-            delay,
+            delay: this.frame_delay
```

Comment:
This function is still accepting a `delay` parameter which is no longer being used.

Otherwise, looks good, thanks!

Reply:
I've removed the unused parameter.